app/views/admin/users/show/profile/actions/_banish.html.erb

```diff
@@ -1,21 +1,14 @@
 <div id="banish-for-spam">
   <div class="flex flex-col gap-4">
-    <% if @user.comments.where("created_at < ?", 100.days.ago).empty? && @user.created_at < 100.days.ago %>
-      <p>This action is irreversible.</p>
-      <p>Once banished, we will delete all content created by <%= @user.name %> and change their username to @spam_###.</p>
-      <p>Be careful with this action.</p>
```

Comment:
Is this code actually needed? Before the copy was changed here, the only difference between the `if` and `elsif` blocks was the copy -- no matter the age of the user or their comments, the action appeared to be the same. If I'm overlooking something or missing context, please let me know, as I'm happy to revert this if need be!

As an aside, I tested that this still works (shows the correct copy dependent upon a user's role (Admin vs. Super Admin vs. Support Admin) and banishes the user) as expected with the changes to the conditional that I've made and it appears to.

Reply:
I think we can consolidate this into one statement like you did because as you mention the contents of these statements are the same. 

However, I suspect that we need to keep all the conditions in place as well. It seems like we don't banish `not-new` users. We determine if the user is `not-new` by the condition  `@user.comments.where("created_at < ?", 100.days.ago).empty? && @user.created_at < 100.days.ago`


We'd have to do something like:
`if (@user.comments.where("created_at < ?", 100.days.ago).empty? && @user.created_at < 100.days.ago) || current_user.super_admin? || current_user.support_admin? `

Reply:
Ahhh, gotcha! That makes a ton of sense. I've made this change. 😊 

Thanks so much for the input, @Ridhwana!